src/main/java/com/exasol/containers/wait/strategy/LogFileEntryWaitStrategy.java

```diff
@@ -67,4 +67,8 @@ protected void waitUntilReady() {
         }
         throw new ContainerLaunchException("Timeout: " + this.detector.describe());
     }
+
+    protected long getWaitTimeOutMilliseconds() {
```

Comment:
Just a suggestion, return TimeUnit.MILLISECONDS, then the returned type already will indicated the metrics of the value.

Reply:
I adopted the style used in the base containers. On a green field I would have used Java's built-in time functions.